linkerd/docs/telemetry.md

```diff
@@ -103,3 +103,30 @@ Key        | Default Value | Description
 ---------- | ------------- | -----------
 sampleRate | _required_    | What percentage of traces to record.
 capacity   | 10            | The maximum number of recent traces to store
+
+## Usage
+
+> Example usage config
+
+```yaml
+telemetry:
+ - kind: io.l5d.commonMetrics
+ - kind: io.l5d.usage
+```
+
+kind `io.l5d.usage`
+
+In order to make improvements and prioritize features, we'd like to gain a
```

Comment:
Not sure if this is the right place but somewhere we should probably stress that this is opt-in and no data will be transmitted until this telemeter is added to the config.